scripts/gen-aliasarr.template.sh

```
@@ -178,6 +178,44 @@ _arr_sanitize_error() {
   printf '%s' "$value"
 }
 
+_arr_clean_ip_payload() {
+  local payload="$1"
+  payload="$(printf '%s' "$payload" | tr -d '\r')"
+  payload="$(_arr_trim "$payload")"
+  printf '%s' "$payload"
+}
+
+_arr_extract_public_ip() {
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this function.

(SC1009)